pipeline/testing/sast/sweep.py

```
@@ -0,0 +1,138 @@
+try:  # 双模式导入：包内 / 独立脚本
+    from pipeline.testing import _yamlmini  # noqa: F401
+    from pipeline.testing.sast import ledger as ledger_mod
+except ImportError:
+    sys.path.insert(0, str(Path(__file__).resolve().parents[3]))
+    from pipeline.testing.sast import ledger as ledger_mod
+
+
+def load_alerts(path):
+    doc = json.loads(Path(path).read_text(encoding="utf-8"))
+    if isinstance(doc, list):
+        # CodeQL REST /repos/{repo}/code-scanning/alerts 原始形状 → 约定形状（机械转换）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)